src/components/Select/ComboBox.test.tsx

```diff
@@ -413,3 +414,18 @@ test('it renders an inverse color * when the field is required', async () => {
     'ChromaSelect-required ChromaSelect-requiredInverse'
   );
 });
+
+test('it renders an icon when icon and tooltipMessage are provided', async () => {
+  const props = getBaseProps();
+  const { findByRole } = renderWithTheme(
+    <ComboBox
+      {...props}
+      icon={IconComponent}
+      tooltipMessage="tooltipMessage"
+      data-testid={testId}
+    />
+  );
+
+  const icon = await findByRole('img', { hidden: true });
```

Comment:
`findByRole('img'` is failing here because there are two SVGs in ComboBox. 

@sternetj @shawnzhu @sam Any ideas on how to be more specific when finding the tooltip icon?

Reply:
I haven't tried it, but based on the [docs](https://testing-library.com/docs/queries/byrole), you might be able to filter by the container. Something like:

```typescript
await findByRole(await findByTestId('ComboBox'), 'img', { hidden: true })
```

Reply:
@sternetj I couldn't get that to work 🤷‍♂️

Reply:
the `IconComponent` has a testId `iconComponent`, so you can do:

```TSX
import {
  IconComponent,
  testId as iconTestId,
} from '../../testUtils/IconComponent';
...
  const { findByTestId } = renderWithTheme(
    <ComboBox
      {...props}
      icon={IconComponent}
      tooltipMessage="tooltipMessage"
      data-testid={testId}
    />
  );

  const icon = await findByTestId(iconTestId);
```

Reply:
I like this conversation started by the new jest github actions reporter!

Reply:
✅ 🙏🏻